src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -3227,10 +3227,8 @@ public void T114_String_CreateNew_ReadWrite_Open_ReadAndWrite()
         public void T113_String_CreateNew_Write_Open_Read()
```

Comment:
I think you'll need to either skip or do the old check on desktop.

Reply:
thanks!